src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
It is better to delete the `Main` class before PR
You have a `SalaryInfoTest` class for testing